packages/ui-extensions/docs/surfaces/point-of-sale/reference/examples/navigation-api/navigation-tree.tsx

```diff
@@ -0,0 +1,8 @@
+// You can navigate to any of these three screens since they all exist within the same Navigator.
+return (
+  <Navigator>
```

Comment:
This example seems incomplete compared to its TS counterpart.

Reply:
What do you mean?